### DISCUSSION TITLE
Is it possible to control how many subseparators there are between separators in cartesian chart axis?

### DISCUSSION THREAD

**Discussion by MattiPirinen** [Live-Charts/LiveCharts2#1142] [Q&A] [answered]
It seems that as a default there are three subseparators between separators in cartesian chart axis. Is I would like there to be four of them. Is this possibly currently?

![image](https://github.com/beto-rodriguez/LiveCharts2/assets/23723983/06740bd0-e9a1-41a8-9f34-74c95758c22a)

**Comment by beto-rodriguez** [comment]
This is currently not supported, but at this point it could be easily changed in the source code, this is the line that controls that:\

https://github.com/beto-rodriguez/LiveCharts2/blob/master/src/LiveChartsCore/Axis.cs#L100

changing that number would create more sub-separators.

Why do you think this is relevant? can you please explain your use case? just to have an argument to make this field public

**Reply by MattiPirinen** [reply]
For the user to be able to read precise values from the chart. For example in the picture I added to this post it is a lot more intuitive to read exact values from the line chart if the subseparators of x-axis would be every 100 instead of every 125. Ofc. the user can always just hover over the line series to see values, but If this chart is printed then that is not possible. Please let me know if you are planning of implementing this as property. Otherwise I will probably go with the source code route. 

Thanks in advance

**Reply by beto-rodriguez** [reply, marked as answer]
That sounds fair, I will move this to an issue so we can track the implementation of this feature #1165.
